temporian/core/operators/unary.py

```diff
@@ -182,11 +182,30 @@ def get_output_dtype(cls, feature_dtype: DType) -> DType:
         return feature_dtype
 
 
+class RoundOperator(BaseUnaryOperator):
+    @classmethod
+    def op_key_definition(cls) -> str:
+        return "ROUND"
+
+    @classmethod
+    def allowed_dtypes(cls) -> List[DType]:
+        return [
+            DType.INT32,
+            DType.INT64,
+        ]
+
+    @classmethod
+    def get_output_dtype(cls, feature_dtype: DType) -> DType:
+        return feature_dtype
```

Comment:
These look off:
- `allowed_dtypes` specifies the types that the operator can consume - should be DType.FLOAT32 and DType.FLOAT64
- `get_output_dtype` returns the output dtype, given the input dtype - in this case it should return the same type as received (I said it should return ints in the issue, edited it, needs to return floats because they have a much larger range than ints)